extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -353,97 +329,88 @@ export abstract class AzureAuth implements vscode.Disposable {
 		}
 	}
 
-	protected async getSubscriptions(account: AzureAccount): Promise<Subscription[]> {
-		interface SubscriptionResponse { // https://docs.microsoft.com/en-us/rest/api/resources/subscriptions/list
-			subscriptionId: string
-			tenantId: string
-			displayName: string
+	//#endregion
+
+	//#region token management
+	private async saveToken(tenant: Tenant, resource: Resource, accountKey: azdata.AccountKey, { accessToken, refreshToken, expiresOn }: OAuthTokenResponse) {
+		const msg = localize('azure.cacheErrorAdd', "Error when adding your account to the cache.");
+		if (!tenant.id || !resource.id) {
+			Logger.pii('Tenant ID or resource ID was undefined', tenant, resource);
+			throw new AzureAuthError(msg, 'Adding account to cache failed', undefined);
```

Comment:
Seems like you're using this message in multiple different places so maybe it can be added to the localizedConstants file

Reply:
Will probably address this in the future.